queries/account-data.rq

```diff
@@ -16,6 +16,7 @@ WHERE {
   ?statement ?propValuePredicate ?value .
 
   MINUS { ?statement pq:P582 [] }
+  MINUS { ?statement pq:P2241 [] }
```

Comment:
```suggestion
  MINUS { ?statement wikibase:rank wikibase:DeprecatedRank }
```

I think the idea of removing those with deprecated rank is great, but the solution is a roundabout way to do it. 
If someone changes rank to deprecated but forgets to add the reason, then this will still not catch it. Better would be to remove any statement with the deprecated rank.